usaspending_api/etl/management/commands/load_transactions_in_delta.py

```diff
@@ -184,24 +223,133 @@ def source_subquery_sql(self, transaction_type):
         else:
             raise ValueError("Invalid value for 'transaction_type'; must select either: 'fabs' or 'fpds'")
 
-        select_cols = []
-        omitted_cols = ["transaction_id"]
-        for col in filter(lambda x: x.silver_name not in omitted_cols, col_info):
-            if col.is_cast:
-                select_cols.append(
-                    f"CAST({bronze_table_name}.{col.bronze_name} AS {col.delta_type}) AS {col.silver_name}"
+        if self.etl_level in ("transaction_fabs", "transaction_fpds"):
+            select_cols = []
+            additional_joins = ""
+            id_col_name = "transaction_id"
+            omitted_cols = [id_col_name]
+            for col in filter(lambda x: x.silver_name not in omitted_cols, col_info):
+                if col.handling == "cast":
+                    select_cols.append(
+                        f"CAST({bronze_table_name}.{col.bronze_name} AS {col.delta_type}) AS {col.silver_name}"
+                    )
+                elif col.handling == "literal":
+                    # Use col.bronze_name directly as the value
+                    select_cols.append(f"{col.bronze_name} AS {col.silver_name}")
+                else:
+                    select_cols.append(f"{bronze_table_name}.{col.bronze_name} AS {col.silver_name}")
+
+        elif self.etl_level == "transaction_normalized":
+            id_col_name = "id"
+            select_cols = [
+                "award_id_lookup.id AS award_id",
+                "awarding_agency.id AS awarding_agency_id",
+                f"""CASE WHEN month(to_date({bronze_table_name}.action_date)) > 9 
+                             THEN year(to_date({bronze_table_name}.action_date)) + 1
+                         ELSE year(to_date({bronze_table_name}.action_date))
+                    END AS fiscal_year""",
+                "funding_agency.id AS funding_agency_id",
+            ]
+
+            if transaction_type == "fabs":
+                select_cols.extend(
+                    [
+                        # business_categories
+                        f"get_business_categories_fabs({bronze_table_name}.business_types) AS business_categories",
+                        # funding_amount
+                        f"""
+                        CAST(COALESCE({bronze_table_name}.federal_action_obligation, 0)
+                                + COALESCE({bronze_table_name}.non_federal_funding_amount, 0)
+                             AS NUMERIC(23, 2)) AS funding_amount
+                        """,
+                    ]
                 )
+                map_col_info = copy.copy(FABS_TO_NORMALIZED_COLUMN_INFO)
             else:
-                select_cols.append(f"{bronze_table_name}.{col.bronze_name} AS {col.silver_name}")
+                fpds_business_category_columns = copy.copy(fpds_boolean_columns)
```

Comment:
Can you make the `else` above this line into an `elif` then add an `else` to catch bad `transaction_types` at the bottom, similar to what you have on line 224:
>  raise ValueError("Invalid value for 'transaction_type'; must select either: 'fabs' or 'fpds'")

Reply:
I see that you're already doing this check in `transaction_normalized_merge_into_sql()`, so this is optional. I still like to be explicit about what condition leads to this code path for the ease of readability though. Maybe no need for the final `else`, but keep the `elif`

Reply:
ok